File: Python/Problem-8.py
# Write a Program To Find The Leap Year
year = int(input("Enter Your Years: "))
if year % 400 == 0 and year % 100 == 0:
  print(f"This Is {year} Leap Year")
elif year % 4 == 0 and year % 100 != 0:
  print(f"This is {year} Leap Year ")
else:
  print("Not Leap Year")

File: JavaScript/Problem-8.js
// # Write a Program To Find The Leap Year
const prompt = require('prompt-sync')()

var year = parseInt(prompt("Enter Any Year: "))
if (year % 400 === 0 && year % 100 === 0){
  console.log(`${year} is a Leap Year`)
}
if (year % 4 === 0 && year % 100 !== 0){
  console.log(`${year} is a leap Year`)
}else{
  console.log(`${year} is not a Leeap Year`)
}